sideboard/lib/sa/__init__.py

```diff
@@ -80,9 +80,9 @@ def process_bind_param(self, value, dialect):
             return str(value)
         else:
             if not isinstance(value, uuid.UUID):
-                return '%.32x' % uuid.UUID(value)
+                return uuid.UUID(value).hex
```

Comment:
It's not super-clear what this change is for (although it certainly looks more idiomatic at first glance).